src/book.js

```diff
@@ -0,0 +1,12 @@
+export default function book(){
+    return `
+        <h1>Book</h1>
+            <ol>
+                <li><img src = "" alt = ""></li>
+                <li>Author</li>
+                <li>Price</li>
+                <li>Selling Points</li>
+            </ol>
```

Comment:
Make sure that all siblings (in this case `h1` and `ol`) are indented to the same level.